src/cljs/org/broadinstitute/firecloud_ui/page/workspace/workspace_method_confs.cljs

```diff
@@ -36,10 +38,126 @@
+                      {:header-component (header "Type") :starting-width 100
+                       :cell-renderer (fn [row-num conf] (cell (conf "rootEntityType")))}
+                      {:header-component (header "Workspace Name") :starting-width 160
+                       :cell-renderer (fn [row-num conf] (cell (str ((conf "workspaceName") "namespace") ":"
+                                                                 ((conf "workspaceName") "name"))))}
+                      {:header-component (header "Method") :starting-width 210
+                       :cell-renderer (fn [row-num conf] (cell (str ((conf "methodStoreMethod") "methodNamespace") ":"
+                                                                 ((conf "methodStoreMethod") "methodName") ":"
+                                                                 ((conf "methodStoreMethod") "methodVersion"))))}
+                      {:header-component (header "Config") :starting-width 290
+                       :cell-renderer (fn [row-num conf] (cell (str ((conf "methodStoreConfig") "methodConfigNamespace") ":"
+                                                                 ((conf "methodStoreConfig") "methodConfigName") ":"
+                                                                 ((conf "methodStoreConfig") "methodConfigVersion"))))}
+                      {:header-component (header "Inputs") :starting-width 200
+                       :cell-renderer (fn [row-num conf] (cell (stringify_map (conf "inputs"))))}
+                      {:header-component (header "Outputs") :starting-width 200
+                       :cell-renderer (fn [row-num conf] (cell (stringify_map (conf "outputs"))))}
+                      {:header-component (header "Prerequisites") :starting-width 200
+                       :cell-renderer (fn [row-num conf] (cell (stringify_map (conf "prerequisites"))))}]
+            :data (:method-confs props)
+            :row-props (fn [row-num conf]
+                         {:style {:fontSize "80%" :fontWeight 500
+                                  :paddingTop 10 :paddingBottom 7
+                                  :backgroundColor (if (even? row-num) (:background-gray style/colors) "#fff")}})})])])})
+
+
+(defn- modal-import-background [state]
+  {:backgroundColor "rgba(82, 129, 197, 0.4)"
+   :display (when-not (:import-overlay-shown? @state) "none")
+   :overflowX "hidden" :overflowY "scroll"
+   :position "fixed" :zIndex 9999
+   :top 0 :right 0 :bottom 0 :left 0})
+
+(def ^:private modal-import-content
+  {
+   ;:transform "translate(-50%, 0px)" ORIGINAL
+   :transform "translate(-50%,0px)"
+   :backgroundColor (:background-gray style/colors)
+   :position "relative" :marginBottom 60
+   :top 60
+   ;:left "50%"  ORIGINAL
+   :left "50%"
+   ;:width 500 ORIGINAL
+   :width "75%"
+   })
+
+
+(defn- render-import-overlay [state refs]
+  (let [clear-import-overlay (fn []
+      (common/clear! refs "wsName" "wsDesc" "shareWith")
```

Comment:
I passed these comments over! sorry.  Also the "ws...." and "share..." I"ll get rid of too.